README.md

```diff
@@ -38,38 +38,48 @@ Xud uses [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/). You
 
 ## Command-Line Interface
 
+Spawning a new `xud` process
+
+```bash
+~/xud/bin $ ./xud 
+Options:
+  -r, --rpc.port                                        [number] [default: 8886]
+  -p, --p2p.port                                        [number] [default: 8885] 
+```
+
+Interacting with an `xud` process, identified by his rpc port
 ```bash
 ~/xud/bin $ ./xucli --help
 xucli [command]
 
 Commands:
-  xucli connect <host> [port]                   connect to an xu node
-  xucli getinfo                                 get general info from the xud node
-  xucli getorders                               get orders from the orderbook
-  xucli getpairs                                get orderbook's available pairs
-  xucli placeorder <pairId> <price> <quantity>  place an order
-  xucli shutdown                                gracefully shutdown the xud node
-  xucli tokenswap <identifier> <role>           perform a raiden token swap
+  xucli connect <p2p_host> [p2p_port]            connect to an xu node
```

Comment:
I actually think this may have been more clear as original, particularly to distinguish this from the `p2p.port` for `xud` above, which is the port on which to listen on for incoming connections.